tests/models/test_modeling_common.py

```diff
@@ -1080,7 +1080,7 @@ def test_cpu_offload(self):
         torch.manual_seed(0)
         base_output = model(**inputs_dict)
 
-        model_size = compute_module_persistent_sizes(model)[""]
+        model_size = compute_module_sizes(model)[""]
```

Comment:
follow up fix for this https://github.com/huggingface/diffusers/pull/10340/files#r1895134336